.github/workflows/ci-external.yml

```diff
@@ -42,9 +42,7 @@ jobs:
         run: |
           wget -qO- get.nextflow.io | bash
           sudo mv nextflow /usr/local/bin/
-      - name: Run pipeline with NetMHC
-        run: |
-          nextflow run ${GITHUB_WORKSPACE} -profile test_netmhc,docker --outdir ./results
+        # TODO: Add netmhciipan test
```

Comment:
Resolvethis TODO

Reply:
For this I need the help of @christopher-mohr (or you?). I don't have the rights to add / update secrets in github and netmhc binaries are encrypted on the nf-core testdata repo and can only be decrypted by gh secrets

Reply:
I don't think that I have permissions for this unfortunately

Reply:
For me its fine in a separate PR. Its important that it needs to be NetMHCpan-4.1b and NetMHCIIpan-4.3e (regarding checksums)